babel.config.js

```diff
@@ -15,5 +15,6 @@ module.exports = {
   plugins: [
     "@babel/plugin-transform-async-generator-functions",
     "@babel/plugin-transform-class-properties",
+    "@babel/plugin-transform-private-methods",
```

Comment:
Is this really necessary? I don't like the idea of adding new libraries to our lib before lavamoat is fully implemented

Reply:
This is not a new library, if you check the package-lock you will see no new dependency for babel.
This is just to allow babel to convert methods starting with `#` into private methods.

Internally it works by creating a new function on the module and calling it inline when called inside the class.

I can remove this if I rename all private methods from `#method` to `_method`, what do you think?

Reply:
If this is not installing anything else, I see no problem with using it